### PR TITLE
Resprites some of the Stirrup socks and leotard, adds digi variants to the stirrup socks

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

Tin, adds digitigrade variants to the socks where there weren't. Adds a new one, resprites them to use the updated GAGS palette. I'll fix the rest at a later date.
Screenshots of the digitigrade variants
image
image
image
image

## How This Contributes To The Skyrat Roleplay Experience

Updated sprites.

## Changelog

:cl:
imageadd: Thigh high and knee high stirrup socks have been resprited, and have had digitigrade variants added.
/:cl:

### CODE REVIEW

**Comment by Ebin-Halcyon** [comment]
Honestly forgot I added the leotard in this too lol.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -331,6 +331,11 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
 	icon_state = "leggings-stir"
 	use_static = null
 
+/datum/sprite_accessory/socks/leggings/stirrups/gym
+	name = "Leggings - Stirrups black w/ stripe"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
	name = "Leggings - Stirrups black with stripe"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Is it normal that the side sprites for the stirrup socks (especially for digitigrades) are noticeably darker than the front and back sprites?

**Comment by Ebin-Halcyon** [comment]
> Is it normal that the side sprites for the stirrup socks (especially for digitigrades) are noticeably darker than the front and back sprites?

They're the same palette used on all dirs, It only differs on how and where the colors are used. I'll adjust them a bit to seem 'brighter'.

**Review by GoldenAlpharex — APPROVED** [review]
Alright seems good to me!

**Pull request merged**